### ISSUE TITLE
Issue adding custom token

### ISSUE THREAD

**Issue by alejogranerorequest** [TenKeyLabs/dappwright#253]
**Describe the bug**
Import token: during the setup, when I want to import a custom token just passing only the contract address, the execution fails because it tries to fulfill the token symbol and token decimal fields are disabled for editing and auto-fulfilled when the token address is written. 

@osis , could you skip the steps to write the token symbol and after writing the token address just click on Next?

I realized about this testing in CI and using the debug.

**System:**

- OS version [e.g. 22]
- NodeJs version [e.g v15.8.0]
- dAppwright version [2.5.4]
- testing framework [Playwright]
- testing framework version [1.38.0]

**Comment by alejogranerorequest** [comment]
Or maybe just play with the waits until the Token Symbol and Decimal are fulfilled

**Comment by alejogranerorequest** [comment]
![Uploading Screenshot at Nov 06 15-47-34.png…]()

**Comment by alejogranerorequest** [comment]
<img width="487" alt="Screenshot at Nov 06 15-47-34" src="https://github.com/TenKeyLabs/dappwright/assets/110841882/15bf1cad-2e68-45f1-a81c-d63cbd592ebc">

**Comment by osis** [comment]
yup, seems like they have changed the functionality of this form a bit. Shouldn't be too hard to adjust. Would be a good first issue if someone want to PR something.
